styles.css

```diff
@@ -1,20 +1,149 @@
-.home{
-position: relative;
-width: 375px;
-height: 6968px;
-background: #FFFFFF;
-}
-
-.cover{
-display: flex;
-flex-direction: column;
-align-items: flex-start;
-padding: 120px 24px;
-gap: 12px;
-position: absolute;
-width: 375px;
-height: 590px;
-left: 0px;
-top: 92px;
-background: #1C1A19;
-}
\ No newline at end of file
+
+.home {
+  /* width: 375px; */
+
+  /* height: 6968px; */
```

Comment:
- [ ] Kindly make sure you remove all the commented code lines in your code base to make it more professional and cleaner and also for best practices.

Reply:
ok i delete it